reactMovieApp/src/components/MovieRaiting.js

```diff
@@ -0,0 +1,12 @@
+import './MovieRaiting.css';
+
+function MovieRaiting(props) {
+    return (
+        <div className="raitingr">
+            <p>Raiting: {props.item.vote_average}</p>
+        </div>
+    )
+
```

Comment:
u can use inline return here

like this: 

const MovieRaiting = (props) =>  
        <div className="raitingr">
            <p>Raiting: {props.item.vote_average}</p>
        </div>
  
* no {}
* no 'return'
* nice and clean